src/h3/_cy/geo.pyx

```diff
@@ -136,36 +124,40 @@ cdef class GeoPolygon:
         h3_free(self.gp.holes)
 
 
-def polyfill_polygon(outer, int res, holes=None, bool lnglat_order=False):
-    """ Set of hexagons whose center is contained in a polygon.
+def polygon_to_cells(outer, int res, holes=None):
+    """ Get the set of cells whose center is contained in a polygon.
 
-    The polygon is defined as in the GeoJson standard, with an exterior
-    LinearRing `outer` and a list of LinearRings `holes`, which define any
-    holes in the polygon.
+    The polygon is defined similarity to the GeoJson standard, with an exterior
+    `outer` ring of lat/lng points, and a list of `holes`, each of which are also
+    rings of lat/lng points.
 
-    Each LinearRing may be in clockwise or counter-clockwise order
+    Each ring may be in clockwise or counter-clockwise order
     (right-hand rule or not), and may or may not be a closed loop (where the last
     element is equal to the first).
-    The GeoJSON spec requires the right-hand rule, and a closed loop, but
-    this function will work with any input format.
+    The GeoJSON spec requires the right-hand rule and a closed loop, but
+    this function relaxes those constraints.
+
+    Unlike the GeoJson standard, the elements of the lat/lng pairs of each
+    ring are in lat/lng order, instead of lng/lat order.
+
+    We'll handle translation to different formats in the Python code,
+    rather than the Cython code.
```

Comment:
I'm not sure this is wise for perf. In JS, we delay choosing lat/lng vs lng/lat until we're converting to C data formats, because that allows a single copy operation rather than multiple copy operations. With large/many polygons, copy operations could be expensive.